src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java

```diff
@@ -104,37 +111,99 @@ private void compareReplayResult(CategoryComparisonHolder bindHolder,
                 recordList.stream().collect(Collectors.groupingBy(CompareItem::getCompareOperation));
         Map<String, List<CompareItem>> resultMap =
                 replayResultList.stream().collect(Collectors.groupingBy(CompareItem::getCompareOperation));
+        Set<String> recordMapKeys = recordMap.keySet();
         // todo: use set instead of list
         final List<String> ignoreKeyList = compareConfig.getIgnoreKeyList();
-        for (Map.Entry<String, List<CompareItem>> stringListEntry : recordMap.entrySet()) {
+        for (Map.Entry<String, List<CompareItem>> stringListEntry : resultMap.entrySet()) {
             String key = stringListEntry.getKey();
             boolean ignore = ignoreKeyList.contains(key);
             if (ignore) {
                 continue;
             }
-            List<CompareItem> recordContentList = stringListEntry.getValue();
-            if (resultMap.containsKey(key)) {
-                List<String> recordContentGroupList =
-                        recordContentList.stream().map(CompareItem::getCompareContent).collect(Collectors.toList());
-                List<String> resultContentGroupList =
-                        resultMap.get(key).stream().map(CompareItem::getCompareContent).collect(Collectors.toList());
-                CompareSDK.arraySort(recordContentGroupList, resultContentGroupList);
-                for (int i = 0; i < recordContentGroupList.size(); i++) {
-                    String source = recordContentGroupList.get(i);
-                    String target = resultContentGroupList.get(i);
+            List<CompareItem> resultCompareItems = stringListEntry.getValue();
+            if (recordMap.containsKey(key)) {
+
+                List<CompareItem> recordCompareGroupList = new ArrayList<>();
+                List<CompareItem> resultCompareGroupList = new ArrayList<>();
+                List<CompareItem> missReplayContentList = new ArrayList<>();
+                List<CompareItem> missRecordContentList = new ArrayList<>();
+                List<CompareItem> recordCompareItems = recordMap.get(key);
+
+                arraySort(recordCompareItems, resultCompareItems, recordCompareGroupList, resultCompareGroupList,
```

Comment:
There is no need to do sorting operations here, directly use compareKey to match, and then compare